environment.yml

```diff
@@ -0,0 +1,16 @@
+name: r5py
+
+channels:
+    - conda-forge
+
+dependencies:
+    - python
```

Comment:
Typically, you would want to test against specific versions of Python, i.e. here we would specify e.g. `python=3.9` or `python=3.10`. I would aim to put a bit of control here and also be specific that the library is tested against Python versions 3.8 upwards or something. I would aim to support a bit older versions of Python (i.e. up from 3.8) because people tend to upgrade their environments a bit slowly.

Reply:
How would I specify more than one version here? (never used conda, can’t find anything on the quick)

Reply:
I would anyway carry out tests in a matrix (still have to look into how github workflows look like for that), so the multiple versions get tested anyway - so should we only specify the minimum version r5py requires (so `python>=3.8`)?

Reply:
Basically, you want to create a different environment file for different versions. I am just now creating one environment for Python 10 with developer tools included as well. When writing the GA workflow, there we will define the matrix for testing the package with all different environments. Take a look e.g. how geopandas does it here: https://github.com/geopandas/geopandas/blob/main/.github/workflows/tests.yaml

Reply:
@christophfink Now created a PR for a developer environment file in #28

Reply:
Named the environment as `python_310_dev.yaml`, so it's easy to understand from the file name that this env is for Python 3.10 with dev tools

Reply:
Anyway, I think, this is something you should take over entirely, since I don’t use conda. I’ll close this pull request, as it is redundant, now.